x-pack/plugins/security_solution/public/detections/components/rules/query_bar/index.tsx

```diff
@@ -29,7 +29,7 @@ import * as i18n from './translations';
 export interface FieldValueQueryBar {
   filters: Filter[];
   query: Query;
-  saved_id?: string;
+  saved_id?: string | null;
```

Comment:
Can `save_id` still be `undefined`?

Reply:
For form it shouldn't be undefined, you're right. I put it back for debugging purpose, but right now removed it